### DISCUSSION TITLE
Blind Level Action "hoch runter Button" vertauschen

### DISCUSSION THREAD

**Discussion by backlink-market** [Zefau/ioBroker.jarvis#2099] [Q&A]
Hi,

Bei mir fahren die Rolläden beim Pfeil nach unten hoch und Pfeil nach oben fahren sie runter.
Wie vertausche ich die?

Habe Somfy Tahoma Adapter den Datenpunkt ausgewählt:tahoma.0.devices.Büro_Rechts.states.core:TargetClosureState

Danke

**Comment by backlink-market** [comment]
Hab mir nun die Pro gekauft und mit folgendem CSS Custom Code gelöst:
`.jarvis-widget-[WIDGETID] .mdi-chevron-up {content: url('https://api.iconify.design/mdi/chevron-down.svg');}
 .jarvis-widget-3168e856-e6bc-4411-[WIDGETID] .mdi-chevron-down {content: url('https://api.iconify.design/mdi/chevron-up.svg');}`

gibts ne bessere lösung? Kann man das icon auch local abrufen?

**Comment by mcuiobroker** [comment]
Hast du mal Änderung in den Datenpunkt-Eigenschaften im level DP versucht?
`{"on":100,"off":0}` -> `{"on":0,"off":100}`

**Reply by backlink-market** [reply]
ja, hat nicht geholfen
